optuna/samplers/_tpe/sampler.py

```diff
@@ -94,6 +100,29 @@ def objective(trial):
             study = optuna.create_study(sampler=TPESampler())
             study.optimize(objective, n_trials=10)
 
+    .. note::
+        For `v2.9.0 <https://github.com/optuna/optuna/releases/tag/v2.9.0>`_ or later,
+        :class:`~optuna.samplers.MOTPESampler` is deprecated and this sampler should be used
+        instead. The following code shows how you apply this sampler to a multi-objective task:
```

Comment:
It might be confusing to use remark as "this sampler". How about writing down  the name.
```suggestion
        :class:`~optuna.samplers.MOTPESampler` is deprecated and
        :class:`~optuna.samplers.TPESampler` should be used instead. The following code shows
        how you apply :class:`~optuna.samplers.TPESampler` to a multi-objective task:
```

Reply:
Thank you, I confirmed that self-referencing is common in the documentation, so I will take your suggestion.
https://optuna.readthedocs.io/en/stable/reference/optuna.html